networking/network_observability/troubleshooting-network-observability.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+[id="installing-troubleshooting"]
+= Troubleshooting Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-troubleshooting
+
+toc::[]
+
+To assist in troubleshooting a failed Network Observability components, you can perform some troubleshooting actions.
```

Comment:
What about something more generic here ?

```suggestion
To assist in troubleshooting Network Observability issues, you can perform some troubleshooting actions.
```

I just want to avoid any confusion since you can have issues while components appear "Running"

Reply:
Addressed